doc/api/util.md

```diff
@@ -1792,6 +1792,31 @@ console.log(util.stripVTControlCharacters('\u001B[4mvalue\u001B[0m'));
 // Prints "value"
 ```
 
+## `util.styleText(format, text)`
+
+> Stability: 1.1 - Active development
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `format` {string} A text format defined in `util.inspect.colors`.
+* `text` {string} The text to to be formatted.
+
+This function returns a formatted text considering the `format` passed.
+
+```js
+const textRedCollr = util.styleText('red', 'My red message');
+console.log(textRedColor);
+```
+
+`util.inspect.colors` also provides text formats such as `italic`, and
+`underline` and you can combine both:
```

Comment:
I gather that `util.inspect.colors` is something that predates this PR? It feels wrong to have “colors” like `italic` and `underline`.

Reply:
Yes, that's something that exists for quite some time.